python/mxnet/optimizer/optimizer.py

```diff
@@ -781,6 +784,240 @@ def update(self, index, weight, grad, state):
         ftml_update(weight, grad, prev_d, prev_v, prev_z, out=weight,
                     lr=lr, wd=wd, **kwargs)
 
+@register
+class SGDwFastLARS(Optimizer):
```

Comment:
is this different from LBSGD defined here - https://github.com/apache/incubator-mxnet/pull/16122/files#diff-0c893416e9e93fbd94dfaa9fa6c13d67R1022

Currently LBSGD implementation is buggy, maybe this should replace that rather than create SGDwFastLARS?

Reply:
Thank you for quick review.

LB stand for "Large batch" indicating that it will use diverse techniques. As warmup could be implemented directly with lr_scheduler (LARS contrarily to what LBSGD indicate is not a warmup strategy but an optimizer). I think the only reason to use such name would be to have the choice between LARS and LARC optimizers (which will be coming in another PR), or even other techniques for large batch. 

I think it could make sense as both implementation are very similar although I'm not sure it's ergonomic to have the selection of optimizer as an argument of `Optimizer`.

Reply:
yes, i am not suggesting we have a parameter that will switch the optimizer from LARS to LARC. Maybe we could just deprecate( or mark it for deprecation) LBSGD, since it does not work.

Reply:
I think deprecation is the best option, please reviewers thumb up / down this comment for opinion

Reply:
I am in favorof deprecation of `LBSGD`, but still I would prefer a better name - "Fast" in "SGDwFastLARS" does not really make sense (fast compared to what?). Maybe just `LARSSGD` or something like that?

Reply:
We could also just call them LARC or LARS.

As a point of reference, TF calls it LARSOptimizer:
https://www.tensorflow.org/api_docs/python/tf/contrib/opt/LARSOptimizer

Reply:
I renamed it LARS, and also remove the "lars" redundant prefix from `eta` and `eps` let me know if it's fine like that